feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt

```diff
@@ -172,6 +182,50 @@ class SessionDetailFragment : DaggerFragment() {
         }
     }
 
+    private fun drawSessionDescription() {
```

Comment:
How about change method name to setupSessionDescription?  👀

Reply:
Agree! I will do so.